works.yaml

```diff
@@ -8960,3 +8960,17 @@ nEXT Browser:
   - browser
   - software
   - web
+FarmBot:
+  uri:
+  - https://farmbot.io/
+  description:
+  - Farmbot is an open source farming machine.
+  license:
+  - MIT/X11 License
+  - CC0
```

Comment:
If I'm not mistaken, both of these licenses apply, each to a different part of the whole project.  Is that correct?  If so, they should be listed together as all part of the same licensing scheme, like this:

    - MIT/X11 License + CC0

Licenses in separate list items is more for cases where you can choose more one licensing scheme from a list of more than one.

Please let me know if my understanding of the project's license terms is incorrect.  Otherwise, please make the change I suggested to your pull request.